src/routes/task.js

```diff
@@ -47,9 +47,10 @@ router.post('/:task_id', (request, response, next) => {
     console.log(`Error loading user ${currentUser.handle}, or Task id ${task_id}.`);
     throw err
   })
-
-
-
 })
 
+router.get('/all', (request, response, next) => {
+  Promise.all([task.getAll(), user.findAll() ])
+  .then(results => response.json(results))
```

Comment:
Instead of returning all the tasks & users, can you write a new SQL query that fetches the overdue tasks(joined with the user's github handle)?  There is too much work happening on the front end that can just be reduced to a SQL query